examples/notebook_utils.py

```diff
@@ -0,0 +1,14 @@
+import logging
+from pathlib import Path
+import os
+
+
+def set_working_dir() -> str:
+    # Change to project src directory
+    working_dir = Path.cwd()
+    if working_dir.parts[-1] == 'examples':
+        working_dir = Path(Path.cwd()).parents[0] / 'src'
+        os.chdir(working_dir)
+
+    print(working_dir.resolve())
```

Comment:
Can this be removed?

Reply:
Yes. Let's go with existing functionality to run against installed library vs. local. removing notebook_utils.py for now